Dockerfile

```diff
@@ -17,7 +16,7 @@ ARG YUM_REPOSITORY=yum-repository.platform.aws.chdev.org
 
 ENV PATH "${PATH}:${KAFKA_HOME}/bin"
 
-COPY --from=builder /kafka_${SCALA_VERSION}-${KAFKA_VERSION} /opt/kafka_${SCALA_VERSION}-${KAFKA_VERSION}
+COPY --from=builder /opt/kafka_${SCALA_VERSION}-${KAFKA_VERSION} /opt/kafka
```

Comment:
Did this need to change? We need to provide a consistent approach across the 3 Dockerfiles.

Reply:
Noted, thank you.